src/Command/TestDebugCommand.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\AppConsole\Command\TestDebugCommand.
+ */
+
+namespace Drupal\AppConsole\Command;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Drupal\Component\Serialization\Yaml;
+use Drupal\simpletest\TestDiscovery;
+
+class TestDebugCommand extends ContainerAwareCommand
+{
+
+  protected function configure()
+  {
+    $this
+      ->setName('test:debug')
+      ->setDescription($this->trans('commands.test.debug.description'))
+      ->addArgument('test-id', InputArgument::OPTIONAL,
+            $this->trans('commands.test.debug.arguments.resource-id'))
+      ->addOption('group', '', InputOption::VALUE_OPTIONAL,
+            $this->trans('commands.test.debug.options.group'))
+    ;
+
+    $this->addDependency('simpletest');
+  }
+
+  protected function execute(InputInterface $input, OutputInterface $output)
+  {
+    $test_id = $input->getArgument('test-id');
+    $group = $input->getOption('group');
+
+
+    $table = $this->getHelperSet()->get('table');
+    $table->setlayout($table::LAYOUT_COMPACT);
+
+    if ($test_id) {
+        $this->getTestByID($output, $table, $test_id);
+    } else {
+        $this->getAllTests($output, $table, $group);
+    }
+  }
+
+  /**
+   * @param $output         OutputInterface
+   * @param $table          TableHelper
+   * @param $config_name    String
+   */
+  private function getTestByID($output, $table, $test_id)
+  {
+      $testing_groups = simpletest_test_get_all();
```

Comment:
I think that is better use the service container to get all test availables in Drupal:
container-get(service('test_discovery')->getTestClasses(null)